README.md

```diff
@@ -86,6 +86,8 @@ function App() {
 }
 ```
 
+> Try this out in your browser [here](https://codesandbox.io/s/pk744ggp)!
```

Comment:
Looks great. I don't mind that it differs slightly. I would prefer to keep everything in a single file rather than have multiple files. That way people don't get confused.